doc: update current known issues for 1.6.0 release

b-gent opened the pull request:
Add 1.6.0 label to the existing known issues.

Signed-off-by: Bartosz Gentkowski <bartosz.gentkowski@nordicsemi.no>

NordicBuilder commented:
You can find the documentation preview for this PR at this <a href='http://developer.nordicsemi.com/nRF_Connect_SDK_dev/doc/PR-4840
'>link</a>. It will be updated about 10 minutes after the documentation build succeeds.

*Note: This comment is automatically posted by the Documentation Publishing GitHub Action.*

rugeGerritsen commented on doc/nrf/known_issues.rst:
```diff
@@ -183,7 +184,7 @@ nRF5
 =======
 
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```
NCSDK-9786 has been fixed in NCS 1.6

rugeGerritsen commented on doc/nrf/known_issues.rst:
```diff
@@ -476,7 +477,7 @@ Subsystems
 ============
 
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```
NCSDK-9106 has been fixed in NCS 1.6

rugeGerritsen commented on doc/nrf/known_issues.rst:
```diff
@@ -1007,7 +1008,7 @@ DRGN-15465: Corrupted advertising data when :option:`CONFIG_BT_EXT_ADV` is set
   Instead, use :c:macro:`BT_LE_ADV_CONN`, and if necessary set the device name in the advertising data manually.
 
-.. rst-class:: v1-5-1 v1-5-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0
```
DRGN-15475 has been fixed in NCS 1.6

rugeGerritsen commented on doc/nrf/known_issues.rst:
```diff
@@ -1043,7 +1044,7 @@ DRGN-13921: Directed advertising issues using RPA in TargetA
   **Workaround:** Remove the device address from the resolving list.
 
-.. rst-class::  v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class::  v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```
DRGN-10367 has been fixed in NCS 1.6

rugeGerritsen reviewed (changes requested):
Reviewed MPSL and SoftDevice Controller updates

hakonfam commented on doc/nrf/known_issues.rst:
```diff
@@ -633,15 +634,15 @@ Reconnection issues after bonding
 ==========
 
-.. rst-class:: v1-5-1 v1-5-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0
```
NCSDK-7173 is fixed
```suggestion
.. rst-class:: v1-5-1 v1-5-0
```

hakonfam commented on doc/nrf/known_issues.rst:
```diff
@@ -687,7 +688,7 @@ The combination of nRF Secure Immutable Bootloader and MCUBoot fails to upgrade
 ============
 
-.. rst-class:: v1-5-1 v1-5-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0
```
This will never be fixed as for as I can tell, should it be documented somewhere else?

b-gent replied:
I don't think so, this is the place where people will go to see if they hit a known issue and for the workaround.
So even if it means adding this tag for every release, I would still keep it here.

hakonfam commented on doc/nrf/known_issues.rst:
```diff
@@ -716,7 +717,7 @@ NCSDK-6848: MCUboot must be built from source when included
   **Workaround:** Set :option:`CONFIG_MCUBOOT_BUILD_STRATEGY_FROM_SOURCE` instead.
 
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
```
KRKNWK-7827 is fixed

```suggestion
.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0 v0-4-0 v0-3-0
```

hakonfam commented on doc/nrf/known_issues.rst:
```diff
@@ -828,15 +829,15 @@ Unstable NFC tag samples
 ==============================
 
-.. rst-class:: v1-5-1 v1-5-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0
```
@jhn-nordic this is fixed, right?

jhn-nordic replied:
yes this is fixed

hakonfam commented on doc/nrf/known_issues.rst:
```diff
@@ -678,7 +679,7 @@ nRF Secure Immutable Bootloader and netboot can overwrite non-OTP provisioning d
   Issue related to NCSDK-7982.
 
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```
```suggestion
.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0
```

LuDuda commented on doc/nrf/known_issues.rst:
```diff
@@ -434,7 +435,7 @@ KRKNWK-6073: Potential delay during FOTA
 =====================
 
-.. rst-class:: v1-5-1 v1-5-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0
```
The issue has been fixed.

```suggestion
.. rst-class:: v1-5-1 v1-5-0
```

b-gent commented:
Implemented comments from @rugeGerritsen @hakonfam & @LuDuda

joerchan commented on doc/nrf/known_issues.rst:
```diff
@@ -496,22 +497,22 @@ DRGN-15448: Incomplete bond overwrite during pairing procedure when peer is not
   This can lead to MIC errors during reconnection if the old LTK is used instead.
 
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```
@grochu This issue is fixed now, right?

grochu replied:
That's right, it's fixed for 1.6.0.

grochu replied:
Yes @joerchan, this is fixed for 1.6.0.

b-gent replied:
tag removed

joerchan commented on doc/nrf/known_issues.rst:
```diff
@@ -496,22 +497,22 @@ DRGN-15448: Incomplete bond overwrite during pairing procedure when peer is not
   **Workaround:** Enable the shell after turning on the NUS notifications or block it until turning on the notifications.
 
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```
Fixed for 1.6.

joerchan commented on doc/nrf/known_issues.rst:
```diff
@@ -496,22 +497,22 @@ DRGN-15448: Incomplete bond overwrite during pairing procedure when peer is not
   **Workaround:** Application should use the disconnected callback to handle pairing failed.
 
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0 v1-1-0 v1-0-0
```
Fixed for 1.6

grochu reviewed (changes requested):
One item should be removed in the Bluetooth section.

tomchy commented on doc/nrf/known_issues.rst:
```diff
@@ -358,51 +359,51 @@ KRKNWK-8200: Successful signal on commissioning fail
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
 
 KRKNWK-5535: Device assert if flooded with multiple Network Address requests
   The device could assert if it receives Network Address requests every 0.2 second or more frequently.
```
Add a new line:
KRKNWK-10490: Dead lock in NCP frame fragmentation logic
If the last piece of a fragmented NCP command is not delivered, the receiving side becomes unresponsive to further commands.

**Workaround:** Adjust NCP connection parameters to make sure that at least one of three retransmissions will be successfully received. This behavior can be tested by performing NVRAM operations on the host side.

b-gent replied:
Please open a separate PR for Zigbee known issues

tomchy replied:
Covered in: https://github.com/nrfconnect/sdk-nrf/pull/4930

MarekPorwisz commented on doc/nrf/known_issues.rst:
```diff
@@ -228,19 +229,19 @@ Missing :file:`CMakeLists.txt`
 ======
 
-.. rst-class:: v1-5-1
+.. rst-class:: v1-6-0 v1-5-1
```
This one is fixed for 1.6.0

joerchan commented on doc/nrf/known_issues.rst:
```diff
@@ -934,12 +935,12 @@ DRGN-15064: External Full swing and External Low swing not working
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0 v1-2-1 v1-2-0
 
 DRGN-6362: Do not use the synthesized low frequency clock source
```
@rugeGerritsen @carlopaparo This is perhaps something that should be moved to limitations?

rugeGerritsen replied:
I agree. We already have a corresponding limitations section for the SoftDevice Controller which mentions this issue. It would be nice to have a similar section for MPSL.

tomchy commented on doc/nrf/known_issues.rst:
```diff
@@ -358,51 +359,51 @@ KRKNWK-8200: Successful signal on commissioning fail
-.. rst-class:: v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
 
 KRKNWK-6071: ZBOSS alarms inaccurate
```
May be fixed in 1.6, addressed in: https://github.com/nrfconnect/sdk-nrf/pull/4908

tomchy replied:
PR merged. Issue can be marked as fixed.

b-gent commented:
@tomchy  @maciejpietras  Please open a separate PR for Zigbee known issues. This PR is only meant to verify whether the current issues persist or have been fixed during the release. If all teams started to submit their issues here, that would turn into a massive mess.

b-gent commented on doc/nrf/known_issues.rst:
```diff
@@ -358,51 +359,51 @@ KRKNWK-8200: Successful signal on commissioning fail
   A coordinator with the :ref:`lib_zigbee_shell` component enabled could assert after executing the ``bdb factory_reset`` command.
 
   **Workaround:** Call the ``bdb_reset_via_local_action`` function twice to remove all the network information.
 
```
Please open a separate PR for Zigbee known issues

tomchy replied:
Covered in: https://github.com/nrfconnect/sdk-nrf/pull/4930

b-gent commented on doc/nrf/known_issues.rst:
```diff
@@ -358,51 +359,51 @@ KRKNWK-8200: Successful signal on commissioning fail
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
 
 KRKNWK-5535: Device assert if flooded with multiple Network Address requests
   The device could assert if it receives Network Address requests every 0.2 second or more frequently.
```
Please open a separate PR for Zigbee known issues

tomchy replied:
Covered in: https://github.com/nrfconnect/sdk-nrf/pull/4930

b-gent commented on doc/nrf/known_issues.rst:
```diff
@@ -358,51 +359,51 @@ KRKNWK-8200: Successful signal on commissioning fail
+.. rst-class:: v1-6-0 v1-5-1 v1-5-0 v1-4-2 v1-4-1 v1-4-0 v1-3-2 v1-3-1 v1-3-0
 
 KRKNWK-5535: Device assert if flooded with multiple Network Address requests
   The device could assert if it receives Network Address requests every 0.2 second or more frequently.
```
Please open a separate PR for Zigbee known issues

tomchy replied:
Covered in: https://github.com/nrfconnect/sdk-nrf/pull/4930

b-gent commented:
Addressed comments from @grochu @maciejpietras @tomchy @joerchan @MarekPorwisz 
This one remains unclear -> https://github.com/nrfconnect/sdk-nrf/pull/4840#discussion_r653697027

b-gent commented:
@czeslawmakarski @pdunaj  please have a look at radio driver and nrf desktop issues and let me know if they persist in 1.6.0.

b-gent commented:
@carlescufi  I think we have enough approvals, this should be good to go

b-gent commented:
Removing @Vge0rge as reviewer as crypto is unaffected by this PR.
